docs/support.md

```diff
@@ -0,0 +1,120 @@
+# Currents Support & Maintenance Policy
+
+## Introduction
+
+This document outlines the support and maintenance policy for Currents. It is intended for customers deploying Currents in self-managed (on-prem) environments. The goal is to clearly define our responsibilities, provide guidance on components we support directly, and set expectations for areas where support is advisory or excluded.
+
+This document may be shared with customers and incorporated into future support material and onboarding guides.
+
+---
+
+## System Components & Support Scope
+
+Currents components fall into three categories:
+
+1. **Core (Fully Supported)**
+2. **Advisory Support**
+3. **Excluded (No Support)**
+
+---
+
+### 1. Core Components (Fully Supported)
+
+Once Currents is deployed and the correct setup is confirmed, Currents is responsible for the reliable and performant operation of the core services. This includes:
+
+- **Currents services running within the Kubernetes (K8S) cluster**
+- **Helm chart** configuration, parameters, and definitions
+- **Database configuration**:
+  - Indexes and query performance
+  - Data integrity and settings (e.g., backups, rollover policies)
```

Comment:
We can advise on backups, (and we should add steps for doing so for the example mongo using mongodump, but anything beyond mongodump will be specific to what software they used for installing mongo. our examples use the community operator which does not have built-in backups, but it is fairly easy to swap to the enterprise version which does, or to percona's mongodb operator which does.

Though likely jus setting up disk snaptshots for the mongodb volume is easiest, but once again is something we can advise on, but can't facilitate directly.

@agoldis and what did you mean by rollover policies? For the mongo replicas?

Reply:
Moved to advisory support

Reply:
@twk3 ES rollover policies are part of our ES index settings